### PR TITLE
Keep the track of current path for types "shouldPopUntil" and "pushNamedAndRemoveUntil".

### PR DESCRIPTION
**Pull request by 0rangeFox**
This pull request aims to maintain the paths whenever it is moved via the methods said in the title, and to remove the previous ones.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: flutter_redux_navigation
 description: A simple reactive navigation middleware for Flutter's redux library.
-version: 0.7.0
+version: 0.7.1
```
Comment (esarbanis) [inline pubspec.yaml]: This is done by the maintainer, please revert

**Review by esarbanis — CHANGES_REQUESTED** [review]
Please revert all changes on the imports and formatting, it is really difficult to follow what has changed. Make sure to only include changes appropriate to the PR scope.

Can you also include a couple of tests?

**Comment by 0rangeFox** [comment]
Why I should revert the imports? This way everything works properly and keeps things more organized to distinguish the differences between what is in the current package with the outside package. About formatting, I have tried my best for better understandability, I think it is very simple and understandable.

**Comment by esarbanis** [comment]
@0rangeFox changing the imports is ok with me, just not in the same PR and commit where `shouldPopUntil` and `pushNamedAndRemoveUntil` are added. Please create a separate request for this change.

In this project, we use `dart format` for code formatting, so please make sure to use this approach.

Please also add a test for this change.

**Comment by 0rangeFox** [comment]
Ah okay, I get it, about the tests, I haven't worked with that much, but I will try to see what I can do on it.

**Comment by esarbanis** [comment]
Thanks, @0rangeFox !

File: pubspec.yaml (hunk 1 above)
Comment (0rangeFox) [reply pubspec.yaml]: Done.

**Comment by 0rangeFox** [comment]
About testing, I couldn't run it because it gave me an error with the build runner saying the following: `Could not find a file named "pubspec.yaml" in "C:\Users\0rangeFox\AppDataLocal\Pub\Cache\hosted\pub.dartlang.org\_fe_analyzer_shared-28.0.0"`, and I couldn't find anything on the internet related to this, and I think it's that code implementation.

**Comment by 0rangeFox** [comment]
Hey, this has been a long time without receiving an answer, could you tell me something about this?

**Pull request closed without merging**